core/base/src/TApplication.cxx

```diff
@@ -628,20 +630,453 @@ TApplication::EExitOnException TApplication::ExitOnException(TApplication::EExit
    return old;
 }
 
+/////////////////////////////////////////////////////////////////////////////////
+/// The function generates and executes a command that loads the Doxygen URL in
+/// a browser. It works for Mac, Windows and Linux. In the case of Linux, the
+/// function also checks if the DISPLAY is set. If it isn't, a warning message
+/// and the URL will be displayed on the terminal.
+///
+/// \param[in] url <web page to be displayed in a browser
+
+void TApplication::OpenInBrowser(const TString &url)
+{
+   // We check what operating system the user has.
+      #ifdef R_MACOSX
```

Comment:
Move preprocessor liness (`#ifdef`, #elif`, `#else`) to the beginning of the line, please - unindented.